packages/td-tools/src/thing-description.ts

```diff
@@ -278,6 +278,13 @@ export interface OAuth2SecurityScheme extends SecurityScheme {
     scopes?: Array<string>;
 }
 
+export interface TuyaCustomBearerSecurityScheme extends SecurityScheme{
+    scheme: "TuyaCustomBearer";
+    key:string;
+    secret:string;
+    baseUri:string;
+}
+
```

Comment:
I think you mixed up the concepts of `SecurityScheme` and `Credentials`. The `SecurityScheme` contains only information about the Security protocol. In other words, it tells consumers WHAT to do to correctly authenticate with the remote WebThing. On the other hand, `Credentials` contains sensitive information needed by the protocol to successfully authenticate. For example, `username` and `password`. Remember also that the `SecurityScheme` is contained in the TD and therefore is public. I should never put secrets inside a TD. 

Therefore, I suggest moving `key` and `secret` inside a credential object.